src/clr-angular/modal/modal.ts

```diff
@@ -121,6 +119,8 @@ export class ClrModal implements OnChanges, OnDestroy {
   fadeDone(e: AnimationEvent) {
     if (e.toState === 'void') {
       this._openChanged.emit(false);
+    } else if (e.toState === 'false' && isPlatformBrowser(this.platformId) && this.modalTitle) {
```

Comment:
Hmm. So this is now dependent on fadeDone animation callback. But looks like we already had `openChanged` dependent on fadeDone though. I think we could make use of *TODO* comment here to remind us to re-visit this later and decouple these from animation callback.

Reply:
Added a comment, though I don't recall having a discussion about not wanting to avoid animation callbacks so if we want to tackle that refactoring we should backlog a story about that.